src/components/LandingRouteHero/index.css

```diff
@@ -0,0 +1,7 @@
+.mainhero-text{
+    font-size: 1.2rem;
+}
+
+.hero-text{
+    font-size: 0.7rem;
```

Comment:
Try to use px here, orelse font size will decrease to very small value while changing screen size

Reply:
thanks for comment but i'm not changing root font size so now its not require if i adds more responsiveness i will try to check this